lib/conform.ex

```diff
@@ -73,10 +73,30 @@ defmodule Conform do
     end
   end
 
+  defp load_imports(opts) do
+    release_dir = Path.dirname(opts.schema)
+    {:ok, [_ | zip_files]} = :zip.list_dir(("#{release_dir}/app.schema.deps.ez") |> to_char_list)
```

Comment:
Good catch, this should definitely gracefully handle the case where the zip doesn't exist.